resources/windows/wix/project.wxs.erb

```diff
@@ -17,7 +17,10 @@
       Compressed="yes"
       Platform="<%= @platform.architecture %>" />
 
-      <!--  We are discussing a MAINT PR to move the componentfilegroup to seperate file 
```

Comment:
The more I look at this file I am convinced that we do need to create a ```componentgroups.wxs.erb``` file, so either:
1. We leave this comment in or
2. Ticket a task for it (1 pointer)

Have created ticket: [RE-6757](https://tickets.puppetlabs.com/browse/RE-6757)